inference-engine/src/auto_plugin/auto_async_infer_request.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#include <string>
+#include <vector>
+#include <memory>
+#include <map>
+
+#include "auto_async_infer_request.hpp"
+
+namespace AutoPlugin {
+using namespace InferenceEngine;
+
+AutoAsyncInferRequest::AutoAsyncInferRequest(const AutoInferRequest::Ptr&           inferRequest,
+                               const AutoExecutableNetwork::Ptr&      autoExecutableNetwork,
+                               const InferenceEngine::ITaskExecutor::Ptr&    callbackExecutor)
+                               : AsyncInferRequestThreadSafeDefault(inferRequest, nullptr, callbackExecutor)
+                               , _inferRequest(inferRequest) {
+    // this executor starts the inference while  the task (checking the result) is passed to the next stage
+    struct ThisRequestExecutor : public ITaskExecutor {
+        explicit ThisRequestExecutor(AutoAsyncInferRequest* _this_) : _this{_this_} {}
```

Comment:
IMHO. `_this` or `_this_` is misleading.

Reply:
yes, will change it